packages/storybook8/stories/Concepts/Rooms/Docs.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+import RoomsCallSnippetText from '!!raw-loader!./snippets/RoomsCall.snippet.tsx';
+
+<Meta title="Concepts/Rooms" />
+
+# Rooms
+
+Azure Communication Services introduces the concept of rooms for developers secure and organized conversations. Users must receive an invite to room in order to join the call for that room, offering a more secure alternative compared to group calls.
+
+A user is invited to a room using the Rooms API as one of three roles:
+
+- Presenter(default)
+- Attendee
+- Consumer
+
+Each role has different capabilities in a room call. The specific capabilities associated with each role are detailed [here](https://learn.microsoft.com/en-us/azure/communication-services/concepts/rooms/room-concept#predefined-participant-roles-and-permissions).
+
+The ability to manage call access and user roles make Rooms suited for the following call scenarios:
+
+- Virtual visits
+- Online classes
+- Presentations
+- Court trials
+- Debates
+
+## Rooms API
+
+The Rooms API serves the purpose of creating rooms, managing users, and adjusting the lifetime of rooms. It is important to note that the Rooms API is a back-end service that is separate from the UI Library. The following diagram illustrates where Rooms API will fit in your architecture.
+
+<img src="images/rooms-architecture.png" alt="Rooms architecture" />
+
+A JavaScript quickstart for using the Rooms API is detailed [here](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/rooms/get-started-rooms?tabs=windows&pivots=programming-language-javascript). After a room is created and users are added to the room, the CallComposite from the UI Library allows users to join the room call.
+
+### Creating a room
+
+Here is the quickstart [sample code](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/rooms/get-started-rooms?pivots=programming-language-javascript&tabs=windows#create-a-room) for creating a room.
+
+### Add or update participants
+
+Users are added to a room with an assigned role as shown in this quickstart [sample code](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/rooms/get-started-rooms?pivots=programming-language-javascript&tabs=windows#add-or-update-participants) and their user IDs would be used for the Rooms API.
+
+## Joining a room with the CallComposite
+
+The following is a code snippet which shows how to join a room call with the CallComposite.
+
+<Source code={RoomsCallSnippetText} />
+
+Also, you can to try this <a href="./?path=/story/composites-call-joinexistingcall--join-existing-call&args=callType:Rooms" target="_blank">preview sample</a> to join an existing room call by providing the room ID in the call locator.
```

Comment:
```suggestion
Also, you can to try this <a href="./?path=/story/composites-callcomposite--join-existing-call" target="_blank">preview sample</a> to join an existing room call by providing the room ID in the call locator.
```

Reply:
I committed your changes. Thanks for fixing the link!